integration_tests/fixtures/applicationData.json

```diff
@@ -90,7 +90,7 @@
   },
   "type-of-ap": {
     "ap-type": {
-      "type": "standard"
```

Comment:
Soz for the big commit: changing things on the Apply page has a lot of repercussions!